proto/ibc/applications/middleware/fee/v1/tx.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Msg defines the ibc/fee Msg service.
+service Msg {
+  // RegisterCounterPartyAddress defines a rpc handler method for MsgRegisterCounterPartyAddress
+  // RegisterCounterpartyAddress is called by the relayer on each channelEnd and allows them to specify their
+  // counterparty address before relaying This ensures they will be properly compensated for forward relaying since
+  // destination chain must send back relayer's source address (counterparty address) in acknowledgement This function
+  // may be called more than once by a relayer, in which case, latest counterparty address is always used.
+  rpc RegisterCounterPartyAddress(MsgRegisterCounterPartyAddress) returns (MsgRegisterCounterPartyAddressResponse);
+  // EscrowPacketFee defines a rpc handler method for MsgEscrowPacketFee
+  // EscrowPacketFee is an open callback that may be called by any module/user that wishes to escrow funds in order to
+  // incentivize the relaying of the given packet.
+  rpc EscrowPacketFee(MsgEscrowPacketFee) returns (MsgEscrowPacketFeeResponse);
+}
+
+// MsgRegisterCounterPartyAddress is the request type for registering the counter party address
+message MsgRegisterCounterPartyAddress {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string address              = 1 [(gogoproto.moretags) = "yaml:\"address\""];
+  string counterparty_address = 2 [(gogoproto.moretags) = "yaml:\"address\""];
```

Comment:
fix yaml

Reply:
interesting it wasn't caught by compiler

Reply:
Updated.